website/content/en/docs/core-tasks/_index.md

```diff
@@ -0,0 +1,9 @@
+
+---
+title: "Core Tasks"
+linkTitle: "Core Tasks"
+weight: 3
```

Comment:
WDYT? 

```suggestion
title: "Operator SDK user guides by Operator Types"
linkTitle: "Operator Types"
weight: 3
```

Reply:
@jmrodri @jmccormick2001 @fabianvf @asmacdo

Reply:
Also, see that the submenus are very prolix/redundant : 

<img width="309" alt="Screenshot 2020-07-10 at 21 12 02" src="https://user-images.githubusercontent.com/7708031/87197338-409b1180-c2f2-11ea-9a8a-b219103305fe.png">


I would recommend just Ansible, Golang and Helm which would be 

- Documentation
|
----- Operator Types
|
------------ Ansible
|
------------ Helm
|
------------ Golang

Reply:
Or the `linkTitle` can be just `"Building Operators"`?

Reply:
@tengqm's idea seems like it'll drive home the point about what the section is for in first glance. Updating the PR with his suggestion for the `linkTitle` and removing redundancies @camilamacedo86 mentioned.